### ISSUE TITLE
Please, add new emojis owing to recent Twemoji update

### ISSUE THREAD

**Issue by Twilightssuperb** [yulon/emoji-box#2]
Good day. I am so excited with your extension, it's convenient and easy to use. :) Some days ago Twemoji had been updated. Now it includes new emojis implemented in iOS 9.1 and something else:
http://blog.emojipedia.org/twemoji-2-0-emoji-changelog/
Also please keep original emoji order, like Twemoji has.

**Comment by yulon** [comment]
Many thanks for your proposal! I will improve as soon as possible.

**Comment by Twilightssuperb** [comment]
Oh, I just need to notice: seems like Twemoji hasn't release updated collection yet: https://github.com/twitter/twemoji/issues/103 Anyway you should stay tuned.

**Comment by yulon** [comment]
OK :stuck_out_tongue_closed_eyes:

**Comment by Twilightssuperb** [comment]
New assets had been already pushed. 🙌 Please, check Twemoji repo. 😏

**Comment by yulon** [comment]
Sorry, a little busy recently, has been updated :joy:

**Comment by Twilightssuperb** [comment]
Twemoji got update to 2.1 with emojis form Unicode 9.0. Please, check Twemoji Github repo. Should I branch it into new issue?

**Comment by yulon** [comment]
Thanks for reminding, has been updated.

It seems still have to reopen this issue for lazy me :sleeping:

**Comment by Twilightssuperb** [comment]
Another one update has been announced! 😉 
http://blog.emojipedia.org/twemoji-2-2-emoji-changelog/
https://github.com/twitter/twemoji/releases/tag/v2.2.0
Now it includes new gender special emojis. They are so specific and some of them consist of 2 chars. That's why I guess you should separate new emoji from other—maybe, just implement empty placeholders that won't be clickable?
Default layout:
![emoji 2 2 default layout](https://cloud.githubusercontent.com/assets/7080928/18726011/3ebd2850-804b-11e6-8894-72d4af38507a.png)
What I guess (empty placeholders are highlighted with arrows):
![twemoji 2 2 modified layout](https://cloud.githubusercontent.com/assets/7080928/18726094/92fc9e82-804b-11e6-8372-922edebceea5.png)

**Comment by yulon** [comment]
Please forgive me for not quite understand what you mean...
Similar to the separator?

But still updated 😂

**Comment by Twilightssuperb** [comment]
> Similar to the separator?

Yep. The reason was given. It's necessary for new gender special emojis only. You made fast and awesome extension, but there are more than 2,400 emojis in Twemoji. My next idea is that you should divide them into categories (see them at the home page emojipedia.org) through horizontal bar. But I don't know how default Twemoji assets are displayed (moreover, they are not sorted into groups), so it may be cumbersome to rearrange all symbols. One more draft—no horizontal bar, no emoji groups names, looks just like indent:
![twemoji 2 2 divided layout draft](https://cloud.githubusercontent.com/assets/7080928/18748926/3f5811ca-80dd-11e6-87a0-4b49fed6b099.png)
P.S. Oh, I've just glanced at your update. Seems, emojis are already divided, all you need is to type indents.
- after 💎
- after 🌰
- after 🔪
- after 🛰
- after 🌊
- after 🎴
- after 🛒
- after 🔵 or 🏳
- after 🇿🇼

**Comment by yulon** [comment]
OK, I get, this is good advice, I will try to improve, maybe also add a select options for default skin tone and gender. Thank you for feedback!

**Comment by Twilightssuperb** [comment]
Hey, I'm glad to know about some design hacks with new version of your extension. Nevetheless I have 2 remarks—# 1 is Twemoji had been updated two days ago to v.[2.3](https://github.com/twitter/twemoji/releases/tag/v2.3.0). # 2 is a nasty bug: after your extension was updated to 1.2.80... I can't launch it in Chromium-based browsers (based on v.58). Error name: _“Unchecked runtime.lastError while running tabs.executeScript: Cannot access contents of url "data:text/html,chromewebdata". Extension manifest must request permission to access this host.”_
You can see error log in attachment.
[Emoji Box error log #1.txt](https://github.com/yulon/emoji-box/files/1032620/Emoji.Box.error.log.1.txt)

**Comment by yulon** [comment]
1. Has been updated~
2. This error is normal in some special pages, but it should be able to launch normally. Probably because there's a lot of change in the last few versions, automatically update has bug, can you try to reinstall it? 🤔

**Comment by Twilightssuperb** [comment]
Unfortunately I still get this error even after reinstallation. Maybe the reason is inside Chromium 58 engine, I've just tried to reproduce this case on older browser (with Chromium 50) and confirm the same error. So strange...

UPD Damn! Even after I installed extension to **new** browser profile, I still get the same error.

P.S. No gender-specified category in tabs (where are these emojis?), no new emojis from Twemoji 2.3... ☹☹☹☹

**Comment by yulon** [comment]
Emmm... I'll try to improve this error.

Now the emojis are temporarily no gender and skin elements, on the one hand I think the pure expression/emotions should not have these categories, on the other hand because I am lazy. :joy:

**Comment by Twilightssuperb** [comment]
It's hard to implement complete skin tones modifier, but you can place it into own tab. As well as gender-specified chars should have own tab, sepearately of non-gender emojis. To my mind, it's easier way to implement these emojis and to do not burden extension code. 😏
